src/tintx/objects.py

```diff
@@ -238,6 +240,31 @@ def get_object_prop(
             obj_slices = [raw3D[:, ind[0], ind[1]] for ind in obj_index]
             field_max.append(float(np.nanmax(obj_slices)))
             field_mean.append(float(np.nanmean(obj_slices)))
+
```

Comment:
I am not sure, it might be that I don't fully understand the approach or missing some caveats here. But couldn't you simple create the polygons using shaply? Something like this:

```
from shapely.geometry import Polygon

geometry = Polygon(zip(grid1.lon[image1==obj], gird.lat[image1==obj]))
```

Of course then we'd have to make sure that the gird1.lon and grid1.lat are always 2D but that could be done within creation of the grid objects.

Reply:
I'd rather work in the cartesian space of the source data (aeqd, utm, etc) as in geodetic coordinates 

I did it with GeoDataFrame in the first place, but the writeout to hdf5 didn't work then. The shapely Polygon couldn't be serialized for some reason. So I kept it in DataFrame with the JSON representation.

I can change it to GeoDataframe to test it out nevertheless.

Indeed, we could have the user provide the crs-object, and also check if a crs-object is already attached to the Dataset.

Reply:
I gave it a mild thought and think that we should at least give the users the opportunity to provide a crs object. I am thinking here also of Limited area model output which usually comes on rotated pole projections. I mean if the input grid is in geodetic coordinates then I guess the polygon Points should be too. We could also think of providing some functionality for transforming coordinates. Either by letting the users set an input crs and a out crs and convert the crs when the data frame is created or by offering some additional methods in the user interface, not sure what is better. But I probably then we'd also have to make sure that the data is plotted correctly.

Reply:
About saving the data: I see two options, either falling back to geojson as a format, which isn't too bad after all but would probably be a little tricky with the multi index dataframe (maybe not - didn't try) or we make a proper (de-) serialisation in the load/save methods.